css/style.css

```diff
@@ -104,4 +104,93 @@ flex-wrap: wrap;
     background-repeat: no-repeat;
     background-size: cover;
 }
-/***** End hero *****/
\ No newline at end of file
+/***** End hero *****/
+/***** faq *****/
+.reviews{
+    background: var(--light-gray);
+    padding: 50px 0;
+}
+.reviews .flex-container{
+    align-items: center;
+    flex-direction: column;
+}
+.section__header{
+    text-align: center;
+}
+.section__title{
+    color: var(--text-color);
+    font-weight: 700;
+    font-size: 2.5rem;
+    text-transform: capitalize;
+    padding-bottom: 27px;
+}
+.section__desc{
+    color: var(--second-text-color);
+    text-align: center;
+}
+.faq .section__header{
+    margin-bottom: 115px;
+}
+.reviews__cards{
+    display: flex;
+}
+.reviews__cards-card-info{
+    color: var(--second-text-color);
+    font-weight: 600;
+    font-size: 0.875rem;
+}
+.faq__cards-card-info{
+    margin: 5px 0;
+}
+.arrow-icon{
+    width: 9px;
+    height: 16px;
+    fill: var(--primary-color);
+}
+.faq__cards-card{
+    display: flex;
+    margin:  10px ;
+    width: 333px;
+    height: 151px;
+    padding: 25px;
+    border-radius: 9px;
+    -webkit-border-radius: 9px;
+    -moz-border-radius: 9px;
+    -ms-border-radius: 9px;
+    -o-border-radius: 9px;
+    background: var(--light-text-color);
+}
+.faq__cards-card-header{
+    margin: 0 10px;
+}
+.faq__cards-card-body h4{
+font-weight: 700;
+text-align: left;
+color: var(--text-color);
+}
+/***** End faq *****/
+@media screen and (max-width: 1200px) and (min-width:577px){
```

Comment:
you don't need to set min and max for break point, use only value, will help avoid rules duplication on  other breakpoints as well.

Reply:
the mobile size sometimes have completely different icons/images

Reply:
yes that's a regular use case, that is totally doable with the usual approach (one screen size), even if you need to override it in your default styling, or a another media query